src/subcommand/giraffe_main.cpp

```diff
@@ -531,6 +531,20 @@ static std::unique_ptr<GroupedOptionGroup> get_options() {
         "scale for gap scores when chaining",
         double_is_nonnegative
     );
+    chaining_opts.add_range(
+        "rec-penalty-chain",
+        &MinimizerMapper::rec_penalty_chain,
+        MinimizerMapper::default_rec_penalty_chain,
+        "penalty for a recombination when chaining, requires -E (ALPHA)",
+        int_is_nonnegative
+    );
+    chaining_opts.add_range(
+        "rec-penalty-fragm",
```

Comment:
This could match the variable name.
```suggestion
        "rec-penalty-fragment",
```

It could also be `frag`, but I don't think `fragm` is a good abbreviation for "fragment".